examples/cluster/plot_segmentation_toy.py

```diff
@@ -80,8 +81,9 @@
 plt.matshow(img)
 plt.matshow(label_im)
```

Comment:
Both this plotting code and the same lines in the next section (lines 100 and 101) can be replaced to improve the presentation of the images as follows:
```suggestion
fig, axs = plt.subplots(nrows=1, ncols=2, figsize=(10, 5))
axs[0].matshow(img)
axs[1].matshow(label_im)

plt.show()
```